<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wFirstSetup</class>
 <widget class="QMainWindow" name="wFirstSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">wFirstSetup</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lblSelectLanguage">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Select your language:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnEn">
      <property name="text">
       <string notr="true">en</string>
      </property>
      <property name="icon">
       <iconset resource="ressources.qrc">
        <normaloff>:/rec/data/flags/en.svg</normaloff>:/rec/data/flags/en.svg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>36</width>
        <height>36</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnDe">
      <property name="text">
       <string notr="true">de</string>
      </property>
      <property name="icon">
       <iconset resource="ressources.qrc">
        <normaloff>:/rec/data/flags/de.svg</normaloff>:/rec/data/flags/de.svg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>36</width>
        <height>36</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnFr">
      <property name="text">
       <string notr="true">fr</string>
      </property>
      <property name="icon">
       <iconset resource="ressources.qrc">
        <normaloff>:/rec/data/flags/fr.svg</normaloff>:/rec/data/flags/fr.svg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>36</width>
        <height>36</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnIt">
      <property name="text">
       <string notr="true">it</string>
      </property>
      <property name="icon">
       <iconset resource="ressources.qrc">
        <normaloff>:/rec/data/flags/it.svg</normaloff>:/rec/data/flags/it.svg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>36</width>
        <height>36</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>599</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
